<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NUi::CSelectTMDB</class>
 <widget class="QDialog" name="NMediaManager::NUi::CSelectTMDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select TMDB</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="searchForTVShows">
     <property name="text">
      <string>Search TV Shows?</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Season:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="byName">
     <property name="text">
      <string>Search by Name and other Criteria</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Episode:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="resultTitle"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="resultReleaseDate"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="resultTMDBID"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Release Date:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="exactMatchesOnly">
     <property name="text">
      <string>Exact Matches Only?</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TMDB ID:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="byTMDBID">
     <property name="text">
      <string>Search only using TMDB ID</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Episode Title:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="resultExtraInfo"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTreeWidget" name="results">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="resultEpisodeTitle"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="byNameGroupBox">
     <property name="title">
      <string>Search Criteria</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="NSABUtils::CDelayLineEdit" name="searchName"/>
      </item>
      <item row="1" column="4">
       <widget class="NSABUtils::CDelayLineEdit" name="searchReleaseYear"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Season:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Release Year:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>TBDB ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Episode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="NSABUtils::CDelayLineEdit" name="searchTMDBID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Search Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="NSABUtils::CDelaySpinBox" name="searchSeason">
        <property name="specialValueText">
         <string>&lt;ignored&gt;</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="NSABUtils::CDelaySpinBox" name="searchEpisode">
        <property name="specialValueText">
         <string>&lt;ignored&gt;</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="resultSeason"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Extra Info</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="resultEpisode"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="resultsLabel">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSABUtils::CDelayLineEdit</class>
   <extends>QLineEdit</extends>
   <header>SABUtils/DelayLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NSABUtils::CDelaySpinBox</class>
   <extends>QSpinBox</extends>
   <header>SABUtils/DelaySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>byName</tabstop>
  <tabstop>byTMDBID</tabstop>
  <tabstop>exactMatchesOnly</tabstop>
  <tabstop>searchForTVShows</tabstop>
  <tabstop>searchTMDBID</tabstop>
  <tabstop>searchName</tabstop>
  <tabstop>searchReleaseYear</tabstop>
  <tabstop>searchSeason</tabstop>
  <tabstop>searchEpisode</tabstop>
  <tabstop>results</tabstop>
  <tabstop>resultTitle</tabstop>
  <tabstop>resultTMDBID</tabstop>
  <tabstop>resultReleaseDate</tabstop>
  <tabstop>resultSeason</tabstop>
  <tabstop>resultEpisode</tabstop>
  <tabstop>resultEpisodeTitle</tabstop>
  <tabstop>resultExtraInfo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NMediaManager::NUi::CSelectTMDB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NMediaManager::NUi::CSelectTMDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
